contracts/interfaces/IERC165.sol

```diff
@@ -1,24 +1,3 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity >=0.7.0 <0.9.0;
-
-/**
- * @dev Interface of the ERC165 standard, as defined in the
- * https://eips.ethereum.org/EIPS/eip-165[EIP].
- *
- * Implementers can declare support of contract interfaces, which can then be
- * queried by others ({ERC165Checker}).
- *
- * For an implementation, see {ERC165}.
- */
-interface IERC165 {
-    /**
-     * @dev Returns true if this contract implements the interface defined by
-     * `interfaceId`. See the corresponding
-     * https://eips.ethereum.org/EIPS/eip-165#how-interfaces-are-identified[EIP section]
-     * to learn more about how these ids are created.
-     *
-     * This function call must use less than 30 000 gas.
-     */
-    function supportsInterface(bytes4 interfaceId) external view returns (bool);
-}
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
```

Comment:
Interesting to see that we're re-exporting here instead of removing the interfaces altogether. I wouldn't care much about backwards compatibility as this is being shipped along a version bump.

Reply:
Yeah, but tt's not only because backwards compatibility. The idea is to let the contents of contracts/interfaces/*.sol gather in one single place the whole interface set being either used or inherited by contracts within contracts/**.sol.